addons/account/data/account_reports_data.xml

```diff
@@ -11,12 +11,14 @@
             <field name="only_tax_exigible" eval="True"/>
             <field name="column_ids">
                 <record id="generic_tax_report_column_net" model="account.report.column">
-                    <field name="name">NET</field>
+                    <field name="name">Net</field>
```

Comment:
Those changes are not mentioned in the commit and seem unrelated :p I know they aren't; but it should be told. Also, the commit mentions "account_reports" instead of "account" (we're in community here). You should also explain the "why" more than the "what". Here, it's essentially usability improvements (that could be the title of you commit IMO)